cross/arm-softfp/toolchain.cmake

```diff
@@ -0,0 +1,31 @@
+set(CROSS_ROOTFS $ENV{ROOTFS_DIR})
+
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_VERSION 1)
+set(CMAKE_SYSTEM_PROCESSOR armv7l)
+
+## Specify the toolchain
+set(TOOLCHAIN "arm-linux-gnueabi")
+
+add_compile_options(-target ${TOOLCHAIN})
+add_compile_options(-mthumb)
+add_compile_options(-mfpu=vfpv3)
```

Comment:
I am in doubt if we really need to add 'target', 'mthumb' and 'mfpu' options in ' cross/arm-softfp/toolchain.cmake' if they are anyway to be already set in 'src/Native/CMakeLists.txt' regardless of 'cross' or 'native' build.